bin/get-amy.py

```diff
@@ -37,11 +44,39 @@ def main(amy_url, output_file, tags):
         'airports' : fetch_info(amy_url, 'export/instructors.yaml')
     }
 
+    if all_:
+        # Fetch workshops that have all of the specified tags.
+        unique_workshops = fetch_info(amy_url, 'events/published.yaml',
+                                      tags=tags)
+    else:
+        # Fetch workshops for each tag. This is required because by AMY treats
+        # with AND operation, while we want to have OR: "get me workshops that
```

Comment:
English hard. "This is required because by default AMY treats tags with AND operation, while we want to have OR […]".